.github/workflows/release_windows_packages.yml

```diff
@@ -71,143 +71,47 @@ jobs:
     env:
       TEATIME_LABEL_GH_GROUP: 1
+      # TODO(scotttodd): ${OUTPUT_DIR}/${FILE_NAME} for DIST_ARCHIVE
       OUTPUT_DIR: ${{ github.workspace }}/output
-      BUILD_IMAGE: ghcr.io/rocm/therock_build_manylinux_x86_64:main
-      DIST_ARCHIVE: "${{ github.workspace }}/output/therock-dist-linux-${{ matrix.target_bundle.amdgpu_family }}${{ inputs.package_suffix }}-${{ needs.setup_metadata.outputs.version }}.tar.gz"
-      FILE_NAME: "therock-dist-linux-${{ matrix.target_bundle.amdgpu_family }}${{ inputs.package_suffix }}-${{ needs.setup_metadata.outputs.version }}.tar.gz"
+      DIST_ARCHIVE: "${{ github.workspace }}/output/therock-dist-windows-${{ matrix.target_bundle.amdgpu_family }}${{ inputs.package_suffix }}-${{ needs.setup_metadata.outputs.version }}.tar.gz"
+      FILE_NAME: "therock-dist-windows-${{ matrix.target_bundle.amdgpu_family }}${{ inputs.package_suffix }}-${{ needs.setup_metadata.outputs.version }}.tar.gz"
       RELEASE_TYPE: "${{ needs.setup_metadata.outputs.release_type }}"
       S3_TARBALL: "therock-${{ needs.setup_metadata.outputs.release_type }}-tarball"
-      S3_PYTHON: "therock-${{ needs.setup_metadata.outputs.release_type }}-python"
-    strategy:
-      fail-fast: false
-      matrix:
-        target_bundle: ${{ fromJSON(needs.setup_metadata.outputs.package_targets) }}
 
     steps:
       - name: "Checking out repository"
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       - uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38 # v5.4.0
         with:
-          python-version: '3.10'
-      # TODO: We shouldn't be using a cache on actual release branches, but it
-      # really helps for iteration time.
-      - name: Enable cache
-        uses: actions/cache/restore@d4323d4df104b026a6aa633fdb11d772146be0bf # v4.2.2
-        with:
-          path: ${{ env.OUTPUT_DIR }}/caches
-          key: portable-linux-package-matrix-v1-${{ matrix.target_bundle.amdgpu_family }}-${{ github.sha }}
-          restore-keys: |
-            portable-linux-package-matrix-v1-${{ matrix.target_bundle.amdgpu_family }}-
+          python-version: '3.12'
 
-      - name: Install the AWS tool
-        run: ./dockerfiles/cpubuilder/install_awscli.sh
-
-      - name: Fetch sources
+      - name: Print status
         run: |
-          # Prefetch docker container in background.
-          docker pull ${{ env.BUILD_IMAGE }} &
-          ./build_tools/fetch_sources.py --jobs 10
-          wait
+          echo "DIST_ARCHIVE: ${DIST_ARCHIVE}"
+          echo "RELEASE_TYPE: ${RELEASE_TYPE}"
+          echo "S3_TARBALL: ${S3_TARBALL}"
 
-      - name: Build Projects
-        run: |
-          ./build_tools/linux_portable_build.py \
-            --image=${{ env.BUILD_IMAGE }} \
-            --output-dir=${{ env.OUTPUT_DIR }} \
-            -- \
-            "-DTHEROCK_AMDGPU_FAMILIES=${{ matrix.target_bundle.amdgpu_family }}"
```

Comment:
I think this will just be a CMake command on Windows, since there is no need to build under a manylinux dockerfile for portability.